src/store/user/index.ts

```diff
@@ -0,0 +1,30 @@
+import { Module } from 'vuex';
+import { RootStateInterface } from '@/store/types';
+import { UserStateInterface } from '@/store/user/types';
+import { getters } from '@/store/user/getters';
+import { mutations } from '@/store/user/mutations';
+import { actions } from '@/store/user/actions';
+
+export const state: UserStateInterface = {
+  Data: {
```

Comment:
naming conventions:
- Data => data
- registration_token => registrationToken

Reply:
@asteof  Yes but I was planning to send the state, without extra lines. I mean in the request body I can insert all the state **UserStateInterface.data**